esp8266/modules/webrepl.py

```diff
@@ -59,18 +59,16 @@ def start(port=8266, password=None):
     stop()
     if password is None:
         try:
-            import port_config
-            _webrepl.password(port_config.WEBREPL_PASS)
+            import webrepl_cfg
```

Comment:
Why the name change? It will only surprise people who have already worked around the problem of a file with plain-text password being there (e.g. by blacklisting the filename). Now they will need to change the blacklist.